lib/rules/declaration-property-value-no-unknown/index.cjs

```diff
@@ -74,6 +74,8 @@ const rule = (primary, secondaryOptions) => {
 			'text-spacing-trim': 'normal | space-all | space-first | trim-start',
 			'text-wrap-style': 'auto | balance | pretty | stable',
```

Comment:
nit

Maybe put them in this order

1. text-wrap-mode
2. text-wrap-style
3. text-wrap

so that they are grouped.

If `text-wrap` uses previous definitions I suppose it should come after them.

Reply:
yes, make sense! will update